osu.Game.Rulesets.Catch/UI/CatchPlayfield.cs

```diff
@@ -108,9 +111,15 @@ protected override void OnNewDrawableHitObject(DrawableHitObject d)
         private bool checkIfWeCanCatch(CatchHitObject obj) => Catcher.CanCatch(obj);
 
         private void onNewResult(DrawableHitObject judgedObject, JudgementResult result)
-            => CatcherArea.OnNewResult((DrawableCatchHitObject)judgedObject, result);
+        {
+            if (transferToCatcher)
+                CatcherArea.OnNewResult((DrawableCatchHitObject)judgedObject, result);
```

Comment:
while it would be quite convenient if all this did was disable the catching animation, that is not the case. it also disables anything to do with hyperdashing. hyperdash trails are gone, catcher afterimage and red tint are gone. none of this is listed in the OP, so I don't know whether this was intended, and so I have to ask.

Reply:
Probably needs more consideration.